getaround_utils/lib/getaround_utils/mixins/loggable.rb

```diff
@@ -25,6 +25,14 @@ def base_loggable_logger
   end
 
   def loggable(severity, message, payload = {})
+    base_loggable_logger.send(
+      :warn,
+      "Deprecated usagage of GetaroundUtils::Mixins::Loggable.loggable(args*). Please use GetaroundUtils::Mixins::Loggable.loggable_log(args*) instead"
```

Comment:
"usage"

Reply:
I'm nitpicking, but I prefer the more idiomatic way of referring to instance methods with `#` 🙂 :

```ruby
GetaroundUtils::Mixins::Loggable#loggable
```

Reply:
And I'm really nitpicking now, but did you mean `*args`?

Reply:
Thks @watsonbox for all your comments

Reply:
https://github.com/drivy/backend-configs/pull/20